docs/docs/services/qix-engine/access-control.md

```diff
@@ -84,347 +140,378 @@ Example JWT:
+Here, Also, `user.id` will be set to `john-doe` based on the `sub` attribute. `employeeType`, and `custom.country`
+become attributes on `user`. The attributes could be used in a (fictitous) rule like this:
 
-```c
-// Evaluates to `true` if the JWT looks like the example above:
-user.employeeType = "developer" and user.custom.country = "sweden"
+```py
+user.id = "john-doe" and user.employeeType = "developer" and user.custom.country = "sweden" and resource._actions = "*"
 ```
 
-### App
+With the example JWT above, this user is granted access for all actions to all resources.
 
-The document (called App in the rules syntax) context.
+### The `resource` Object
 
-It has no additional attributes.
+A _Resource_ is a generic concept can represent documents, or objects within documents. A Resource carries attributes
+which may be used in rule expressions. In rule expressions the resource being accessed is represented by the `resource`
+object.
 
-### App.Object
+#### Common Attributes
 
-An object inside a document context.
+There are different resource types but all share some common attributes:
 
-Attribute | Description | Example condition
---------- | ----------- | -----------------
-approved | Indicator of whether the object was part of the original document when the document was published. Values: `true` or `false`. | `resource.approved="true"`
-description | The object description. | `resource.description="My custom description"`
-objectType | The object type. | `resource.objectType="field" or resource.objectType="my-generic-object"`
-published | Indicator of whether the object is published. Values: `true` or `false`. | `resource.published="false"`
-app.name | Name of the document that the object is part of. | `app.name="Q3_Report"`
+| Attribute | Description |
+| --------- | ----------- |
+| `resource.id` | The unique identifier for the resource. |
+| `resource.owner` | The owner of the resource. |
+| `resource._resourcetype` | The type of the resource being accessed. Equal to `"App"` or `"App.Object"`. |
+| `resource._actions` | The actions to be granted on the resource. See [Actions](#actions) for more details.|
 
-## Conditions
+#### Type-specific Attributes
 
-**Syntax**
+Depending on what type of resource that is being accessed, `resource` may carry additional attributes. If the resource
+type is `"App"`, there are no additional attributes. If the resource type is `"App.Object"`, `resource` has the
+following additional attributes:
 
-```c
-[resource.resourcetype = "resourcetypevalue"] [OPERATOR] [(((<resource.property = propertyvalue) [OPERATOR (resource.property = propertyvalue)))]
-```
+| Attribute | Description | Example condition |
+| --------- | ----------- | ----------------- |
+| `resource.description` | The object description. | `resource.description = "My custom description"` |
+| `resource._objecttype` | The object type. | `resource._objecttype = "field" or resource.objecttype_ = "my-generic-object"` |
```

Comment:
Switched `_` on the second expression?

Also, maybe double check that both `field` (`GenericField`?) and the developer-defined type (`my-generic-object`) is valid for `_objecttype`.